docs/installation.rst

```diff
@@ -12,7 +12,7 @@ To install EasyReflectometry, run this command in your terminal:
 
 .. code-block:: console
 
-    $ pip install git+https://github.com/easyScience/EasyReflectometryLib.git@master
+    $ pip install git+https://github.com/easyScience/EasyReflectometryLib.git@master[dev]
```

Comment:
Why the dev version?
`pip install git+https://github.com/easyScience/EasyReflectometryLib.git` whould be better